src/Conference.tsx

```diff
@@ -343,6 +343,12 @@ const Conference: React.FC = (props) => {
           if (responseBody.websocketUrl) {
             options.websocketUrl = responseBody.websocketUrl;
           }
+          const turnServerConfig = window.localStorage.getItem('turnServer');
+          if (!!turnServerConfig) {
```

Comment:
Why two exclamation marks (!!) ?

Reply:
When you one ! to any object in Typescript/Javascript the object is parsed to a boolean with a not operator:

For instance:
```js
const test = "123"
console.log(!test) //expected result -> false
```
But if the object tested is null or undefined or, in string case, empty the expression return `true`

For instance:
```js
let test = null
console.log(!test) //expected result -> true
test = undefined
console.log(!test) //expected result -> true
test = ''
console.log(!test) //expected result -> true
```

This way when you add two ! to the sentence, the first act as a normal, not operator, and the second one act as a boolean parser with a not operator.

Then when the code executes `!!turnServerConfig`, it check if the `turnServerConfig` variable is a non-empty, non-null, and non-undefined object.


For instance:
```js
let test = "123"
console.log(!!test) //expected result -> true
test = null
console.log(!!test) //expected result -> false
test = undefined
console.log(!!test) //expected result -> false
test = ''
console.log(!!test) //expected result -> false
```